fe1-web/features/evoting/objects/Election.ts

```diff
@@ -1,5 +1,5 @@
 import { Hash, Timestamp } from 'model/objects';
-import { LaoEvent, LaoEventState, LaoEventType } from 'model/objects/LaoEvent';
+import { LaoEvent, LaoEventState, LaoEventType } from 'features/events/objects/LaoEvent';
```

Comment:
I would already remove this dependency, as long as we implement `LaoEvent` and `LaoEventState`,, we'll be fine 😄 

Regarding `LaoEventType`, we might have to replace that with string constants for now, one defined in each feature.